examples/rgbpp/xudt/2-btc-transfer.ts

```diff
@@ -13,35 +13,26 @@ interface RgbppTransferParams {
 }
 
 const transfer = async ({ rgbppLockArgsList, toBtcAddress, xudtTypeArgs, transferAmount }: RgbppTransferParams) => {
-  const xudtType: CKBComponents.Script = {
-    ...getXudtTypeScript(isMainnet),
-    args: xudtTypeArgs,
-  };
-
-  const ckbVirtualTxResult = await genBtcTransferCkbVirtualTx({
-    collector,
-    rgbppLockArgsList,
-    xudtTypeBytes: serializeScript(xudtType),
-    transferAmount,
+  const { ckbVirtualTxResult, btcPsbtHex } = await buildRgbppTransferTx({
+    ckb: {
+      collector,
+      xudtTypeArgs,
+      rgbppLockArgsList,
+      transferAmount,
+    },
+    btc: {
+      fromAddress: btcAccount.from,
```

Comment:
Please fill the `fromPubkey` prop in the examples as it is required if the `fromAddress` is of `AddressType.P2TR`. Note that `btcAccount.fromPubkey` is undefined if `btcAccount.addressType == AddressType.P2WPKH`. ref: https://github.com/ckb-cell/rgbpp-sdk/blob/develop/examples/rgbpp/shared/btc-account.ts#L27-L52
```suggestion
      fromAddress: btcAccount.from,
      fromPubkey: btcAccount.fromPubkey,
```

Reply:
I added `fromPubkey` to the BTC parameters for two RGB++ transfer examples at https://github.com/ckb-cell/rgbpp-sdk/pull/216/commits/21304feb90d57466c670acbe4e2912fe2726b474